src/test/java/Homework17_PB.java

```diff
@@ -15,7 +16,7 @@ public class Homework17_PB extends BaseTest{
        8. Push your code to a remote repository.
        9. Create a pull request.*/
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

![image](https://user-images.githubusercontent.com/104761482/211192579-1891f622-b9f4-47bb-8d1e-e07ad78c6775.png)